test/src/globTest.ts

```diff
@@ -131,19 +131,26 @@ test.ifDevOrLinuxCi("failed peer dep", () => {
     "test-app-one",
     {
       targets: Platform.LINUX.createTarget(DIR_TARGET),
+      config: {
+        files: ["!pnpm-lock.yaml"],
```

Comment:
`pnpm-lock.yaml` is actually a default excludes now. https://github.com/electron-userland/electron-builder/blob/e9eef0c1c7f73a5edfe3026f044c6278641077cb/packages/app-builder-lib/src/fileMatcher.ts#L18

Not sure what this is testing for, so I'm wondering if we should have a different excludes here

Reply:
Cool, I will delelet it. I didn't add this for any special purpose; I just didn't want to modify the snapshot.